### PR TITLE
add microbiome subdomain

### PR DESCRIPTION
**Pull request by shiltemann**

### CODE REVIEW

**Comment by bgruening** [comment]
Can one of the admins verify this patch?

File: dns.tf [hunk 1]
```diff
@@ -46,6 +46,7 @@ variable "subdomain" {
     "climate.usegalaxy.eu",
     "nanopore.usegalaxy.eu",
     "metabolomics.usegalaxy.eu",
+    "microbiome.usegalaxy.eu",
```
Comment (hexylena) [inline dns.tf:49]: Can you add this to the end of the list please?

Comment (shiltemann) [reply dns.tf]: done

**Comment by hexylena** [comment]
ok to test

**Comment by hexylena** [comment]
it does. huzzah!

**Comment by hexylena** [comment]
add to whitelist

(saskia is trustworthy right)

**Pull request merged**